src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.c

```diff
@@ -523,9 +523,8 @@ s32 EnBomBowlMan_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx**
     return false;
 }
 
-static u64* eyeTextures[3] = { gChuGirlEyeOpenTex, gChuGirlEyeHalfTex, gChuGirlEyeClosedTex };
-
 void EnBomBowlMan_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    static void* eyeTextures[3] = { gChuGirlEyeOpenTex, gChuGirlEyeHalfTex, gChuGirlEyeClosedTex };
```

Comment:
There's no need to put the size explicitly.
```suggestion
    static void* eyeTextures[] = { gChuGirlEyeOpenTex, gChuGirlEyeHalfTex, gChuGirlEyeClosedTex };
```